src/project-model.js

```diff
@@ -0,0 +1,27 @@
+function Project(name, taskList = []) {
+  this.name = name;
+  this.taskList = taskList;
+}
+
+// Create methods of Project: Delete, edit, create new, getProjectByName, validateProjectName,
+
+// Project.prototype.edit = function () {
+// };
+
+
+// class Project {
+//   constructor(name, taskList = []) {
+//     this.name = name;
+//     this.taskList = taskList;|
+//   }
+// }
```

Comment:
- [ ] [REQUIRED] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))